text/0000-add-ember-page-title-addon.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2020-06-24
+- Relevant Team(s): Ember CLI
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+- Authors: Benjamin Jegard, Melanie Sumner, Ricardo Mendes
+
+# Add ember-page-title to the app blueprint
+
+## Summary
+
+Add [`ember-page-title`](https://github.com/adopted-ember-addons/ember-page-title) to the default blueprint for new Ember apps as a way to provide improved out-of-the-box (OOB) accessibility for Ember applications.
```

Comment:
does the addon maintain a history of the titles?

Like, if I were to do a bunch of navigations where some, but not all page-title invocations have `replace=true`, does the title always reflect what is rendered?

Does that mean that the page title is dependent on render order?

What happens if I do?

{{#each this.breadcrumbs |breadcrumb|}}
  {{page-title breadcrumb.name}}
{{/each}}

Reply:
I'm not sure, I'll make a dummy app to test ^^

Reply:
I'll add nested route + breadcrumbs test case to ember-page-title and we'll find out.